R/database.R

```diff
@@ -2685,10 +2687,10 @@ get_manta_sv_by_samples = function(these_samples_metadata,
 #'
 #' @param this_sample_id The single sample ID you want to obtain the result from.
 #' @param these_samples_metadata A metadata table containing metadata for this_sample_id, or sample of interest. This parameter is required.
-#' @param force_lift If TRUE, coordinates will be lifted (if needed) to the selected projection. Default is FALSE. WARNING: if running this function in stand-alone mode, set this parameter to TRUE to ensure that the returned calls are in respect to the requested projection.
+#' @param force_lift If TRUE, coordinates will be lifted (if needed) to the selected projection. Default is FALSE. WARNING: if your code calls this function directly, set this parameter to TRUE to ensure that the returned calls are in respect to the requested projection.
 #' @param min_vaf The minimum tumour VAF for a SV to be returned. Default value is 0.1.
```

Comment:
What does the function do if you run it without setting force_lift = TRUE and you request a projection that is not available as a file already?

Reply:
If this would be done on an individual sample that sample would have an additional column called need_lift and all the rows for that sample set to TRUE, indicating that this sample has not been lifted, and thus the coordinates are not in respect to the selected projection. 

To clarify some potential confusion on how this parameter operates, it is always set to FALSE (default) and this parameter can not be controlled within the by_samples function (hardcodes this parameter to FALSE when internally calling by_sample). This is because anytime you're calling by_samples, liftover will always occur on the subset data frame of sv calls that needs to be lifted (based on the selected projection).

For by_sample, this parameter can be toggled to be TRUE, if so liftover_bedpe will be run on the individual sample. If set to FALSE (default) and the sample in question needs to lifted, an additional column will be added stating this, as detailed above. In addition, if this function is called with this parameter set to TRUE, the user will be suggested to first look for their sample in the desired projection with get_combined_sv.